### DISCUSSION TITLE
Capture consequences of projected loss or damage: Loss of life, livelihood, economic, social fabric/community access, environmental damage and so on – C/T #57

### DISCUSSION THREAD

**Discussion by sreyac** [CriticalAssetManagement/CAMS#139] [General]
Requires further clarification and discussion
--

Part of Asset Inventory: Capture consequences of projected loss or damage

The tool should capture, for each asset, and for each hazard level, the likely impacts of each level of loss or damage and their criticality - High/Medium/Low

**Comment by luke-feeney** [comment]
In short term - links to other critical assets

simple text field to add in free form 'total impact' 

in future some more tooling to make assessment 

GAR - UNDRR survey of risk around the world: assessment to code up as follow up - analytics to give country wide inputs and impacts. 

Bounce of BGC to assess interest @luke-feeney

**Reply by luke-feeney** [reply]
@rpwilliams1958
